pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/gp/BSPKitInfo.java

```diff
@@ -0,0 +1,44 @@
+package org.broadinstitute.dsm.model.gp;
+
```

Comment:
am I the only person who doesn't know what BSP stands for?

Reply:
I don't know too, but it is a server in GP I believe